internal/acceptance/sql_table_test.go

```diff
@@ -72,6 +72,70 @@ func TestUcAccResourceSqlTable_Managed(t *testing.T) {
 	})
 }
 
+func TestUcAccResourceSqlTableWithIdentityColumn_Managed(t *testing.T) {
+	if os.Getenv("GOOGLE_CREDENTIALS") != "" {
+		skipf(t)("databricks_sql_table resource not available on GCP")
+	}
+	unityWorkspaceLevel(t, step{
+		Template: `
+		resource "databricks_schema" "this" {
+			name         = "{var.STICKY_RANDOM}"
+			catalog_name = "main"
+		}
+
+		resource "databricks_sql_table" "this" {
+			name               = "bar"
+			catalog_name       = "main"
+			schema_name        = databricks_schema.this.name
+			table_type         = "MANAGED"
+			properties         = {
+				this      = "that"
+				something = "else"
+			}
+
+			column {
+				name      = "id"
+				type      = "bigint"
+				identity  = "default"
+			}
+			column {
+				name      = "name"
+				type      = "string"
+			}
+			comment = "this table is managed by terraform"
+			owner = "account users"
+		}`,
+	}, step{
+		Template: `
+		resource "databricks_schema" "this" {
+			name         = "{var.STICKY_RANDOM}"
+			catalog_name = "main"
+		}
+
+		resource "databricks_sql_table" "this" {
+			name               = "bar"
+			catalog_name       = "main"
+			schema_name        = databricks_schema.this.name
+			table_type         = "MANAGED"
+			properties         = {
+				that      = "this"
+				something = "else2"
+			}
+			
+			column {
+				name      = "id"
+				type      = "bigint"
+				identity  = "default"
```

Comment:
Can we try making this `always`? Just to test changing the identity column, if that is allowed.

Reply:
Not allowed. Added validation.